scripts/qa_crowbarsetup.sh

```diff
@@ -3093,6 +3093,15 @@ function deploy_single_proposal
 # apply all wanted proposals on crowbar admin node
 function onadmin_proposal
 {
+    local proposals
+    local default_proposals
+
+    default_proposals="
+        nfs_client pacemaker database rabbitmq keystone swift \
+        ceph glance cinder neutron nova `horizon_barclamp` ceilometer heat \
+        manila trove barbican magnum sahara murano tempest"
```

Comment:
Not sure that is a good idea. How it is now they will be deployed (or try to be deployed), and run tempest in it for the gating.

I think that the name `default_proposals` is not accurate, but is more `all_proposals`. We need to iterate in all the proposals to see if the user that it (`want_XXX`) or can be deployed.